test/db/formats/qnx

```diff
@@ -18,10 +18,10 @@ NAME=sections of qnx
 FILE=bins/qnx/test.x
 CMDS=iS
 EXPECT=<<EOF
-paddr      size  vaddr      vsize align perm name     type flags 
------------------------------------------------------------------
-0x00000044 0xf6  0x00000044 0xf6  0x0   ---- LMF_LOAD      
-0x00000152 0x6c3 0x00000152 0x6c3 0x0   ---- LMF_LOAD      
+paddr      size  vaddr      vsize align perm name              type flags 
+--------------------------------------------------------------------------
+0x00000044 0xf6  0x00000044 0xf6  0x0   ---- LMF_LOAD               
+0x00000152 0x6c3 0x00000152 0x6c3 0x0   ---- LMF_LOAD_00000152      
```

Comment:
I wonder, maybe better remove leading zeroes? also use `0x` prefix to mark it's hex.

Reply:
i think is better with them, but yes, i can always remove them.